plasmoid/contents/ui/config/ConfigAppearance.qml

```diff
@@ -7,49 +7,71 @@ import org.kde.plasma.extras 2.0 as PlasmaExtras
 
 ConfigPage {
     id: page
-    
+
     property alias cfg_width: width.value
     property alias cfg_compactLabelMaxWidth: compactLabelMaxWidth.value
     property alias cfg_height: height.value
-    property alias cfg_dropdownvisible: dropdownvisible.checked
-    
-    
+    property alias cfg_d_arrownevervisible: d_arrownevervisible.checked
```

Comment:
Camel case these please (e.g `arrownevervisible` -> `arrowNeverVisible`).

Reply:
You mean `cfg_d_arrownevervisible: d_arrownevervisible` to `cfg_d_arrowNeverVisible: d_arrowNeverVisible`?

Or only `d_arrownevervisible` to `d_arrowNeverVisible`?

Reply:
You mean `cfg_d_arrownevervisible: d_arrownevervisible` to `cfg_d_arrowNeverVisible: d_arrowNeverVisible`?

Or only `d_arrownevervisible` to `d_arrowNeverVisible`?

Reply:
The former, so the variable and the id both follow camel case. (Also applies to the two lines below) 
Thanks.

Reply:
@shocklateboy92 review is needed!